prow/pod-utils/clone/types.go

```diff
@@ -27,6 +27,10 @@ type Record struct {
 	Refs     prowapi.Refs `json:"refs"`
 	Commands []Command    `json:"commands"`
 	Failed   bool         `json:"failed"`
+
+	// FinalSHA is the SHA from ultimate state of a cloned ref
+	// This is used to populate RepoCommit in started.json properly
+	FinalSHA string `json:"finalsha,omitempty"`
```

Comment:
Do we want to camel case the json field name?

Reply:
Wouldn't `final-sha` better fit the rest of our naming? I think we're inconsistent here.

Reply:
sure

Reply:
(looks like we use underscores in most of the places, rather than camel case and dashes.. will update that way... and yes we are not very consistent across the code base :-) )